src/it/java/org/testcontainers/containers/ExasolDockerContainer.java

```diff
@@ -1,23 +1,26 @@
 package org.testcontainers.containers;
 
 import java.time.Duration;
+import java.util.Map;
+
 import static java.time.temporal.ChronoUnit.SECONDS;
 
 import org.testcontainers.containers.wait.strategy.WaitStrategy;
 import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
 
-public class ExasolDockerContainer<SELF extends ExasolDockerContainer<SELF>> extends
-JdbcDatabaseContainer<SELF> {
-  public static final String EXASOL_IMAGE = "exasol/docker-db";
-  public static final String EXASOL_VERSION = "6.2.5-d1";
-  public static final String EXASOL_HOST = "192.168.0.2";
-  public static final Integer EXASOL_PORT = 8888;
+public class ExasolDockerContainer<SELF extends ExasolDockerContainer<SELF>>
+  extends JdbcDatabaseContainer<SELF> {
+  private static final String DEFAULT_EXASOL_VERSION = "7.0.1";
```

Comment:
```suggestion
  private static final String DEFAULT_EXASOL_VERSION = "7.0.2";
```

Reply:
That is quick!
It was not released in the morning :))